Remove redundant if statements

DanVanAtta opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/triplea-game/triplea/782)
<!-- Reviewable:end -->

ron-murhammer commented on src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java:
```diff
@@ -71,11 +71,8 @@ public static boolean isAirborneMove(final GameData data) {
-      } else {
-        isAirborneMove = false;
-      }
+      } else
```
Missing bracket.

ron-murhammer commented on src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java:
```diff
@@ -144,16 +141,9 @@ public static boolean isFireRockets(final GameData data) {
-        isFireRockets = false;
-      }
+        isFireRockets = isCombatDelegate(data);
+      } else
```
Missing bracket.

ron-murhammer commented on src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java:
```diff
@@ -179,11 +169,8 @@ public static boolean isRepairUnits(final GameData data) {
-        } else {
-          isRepairUnits = false;
-        }
+        } else
```
Missing bracket.

ron-murhammer commented on src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java:
```diff
@@ -201,11 +188,8 @@ public static boolean isGiveBonusMovement(final GameData data) {
-      } else {
-        isBonus = false;
-      }
+      } else
```
Missing bracket.

ron-murhammer commented on src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java:
```diff
@@ -229,11 +213,8 @@ public static boolean isRemoveAirThatCanNotLand(final GameData data) {
-      } else {
-        isRemoveAir = false;
-      }
+      } else
```
Missing bracket.

ron-murhammer commented on src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java:
```diff
@@ -307,11 +288,8 @@ public static boolean isResetUnitStateAtEnd(final GameData data) {
-      } else {
-        isReset = false;
-      }
+      } else
```
Missing bracket.

ron-murhammer commented on src/games/strategy/triplea/delegate/GameStepPropertiesHelper.java:
```diff
@@ -327,11 +305,8 @@ public static boolean isBid(final GameData data) {
-      } else {
-        isBid = false;
-      }
+      } else
```
Missing bracket.

ron-murhammer commented on src/games/strategy/triplea/delegate/Matches.java:
```diff
@@ -135,11 +135,8 @@ public boolean match(final Unit unit) {
-      } else {
-        return false;
-      }
+      } else
```
Missing bracket.

ron-murhammer replied:
This bracket still appears to be missing.

RoiEXLab commented:
@ron-murhammer about the Missing Bracket thing... I think you're wrong...In most cases I'm seeing 5 brackets being removed and 1 being added...

DanVanAtta commented:
PR updated, merge conflicts resolved.

@RoiEXLab  - it does appear the auto-simplify done by intelliJ on this one did use simplified 'else' blocks, that did not have the curly braces.

I added in the curly braces, but also there was a lot of opportunity to simplify the if/else blocks.

RoiEXLab commented:
@DanVanAtta Time to re-self-assign...
boolean isFireRocktes in GameStepPropertiesHelper.java needs initialization

RoiEXLab commented:
@DanVanAtta Uh oh. Seems like you broke it ^^
A test is failing...

DanVanAtta commented:
@RoiEXLab  - I"m getting this on the latest master. Would you mind taking a look to see if you can repro?

```
games.strategy.engine.lobby.server.login.LobbyLoginValidatorTest > testAnonymousLoginBadName FAILED
    java.lang.IllegalArgumentException: You have an invalid MAC address!
        at games.strategy.net.MacFinder.getHashedMacAddress(MacFinder.java:24)
        at games.strategy.engine.lobby.server.login.LobbyLoginValidatorTest.testAnonymousLoginBadName(LobbyLoginValidatorTest.java:70)
```

RoiEXLab commented:
@DanVanAtta Although it's 1 am here, and I was just about going to sleep, I took a look at this one...
I cannot reproduce this Test-Failure on master...
When testing your branch, I get the following failures/errors:
[Gist](https://gist.github.com/RoiEXLab/8b2014e7cf2261053a337edc9926e414)

DanVanAtta commented:
@ron-murhammer  this is clean now. The refactor in this PR broke some tests, and I ran it on a machine that revealed additional test failures when running tests on a laptop via wifi. Both issues have been addressed, and for this PR - I removed the bad refactor and added instead a simple commit that added in curly braces

ron-murhammer commented:
@DanVanAtta Looks good but I think you missed 1 bracket :)

DanVanAtta commented:
fixed it (c70afa2) - now updated
